packages/containers/src/DeleteResource/DeleteResource.connect.js

```diff
@@ -1,4 +1,15 @@
 import { cmfConnect } from '@talend/react-cmf';
+import Immutable from 'immutable';
 import Container from './DeleteResource.container';
 
-export default cmfConnect({ componentId: 'DeleteResource' })(Container);
+export function mapStateToProps(state, ownProps) {
+	if (ownProps.resourceType) {
+		const resource = state.cmf.collections
+			.get(ownProps.resourceType, new Immutable.Map())
+			.find(currentResource => currentResource.get('id') === ownProps.routeParams.id);
+		return { resource };
+	}
+	return {};
+}
+
+export default cmfConnect({ componentId: 'DeleteResource', mapStateToProps })(Container);
```

Comment:
componentId: 'DeleteResource' => componentId: ownProps => ownProps.id || ownProps.componentId
Right now the componentId is hardcoded.